### PR TITLE
Implement vectorization of #existing and inverse masks

### PR DESCRIPTION
**Pull request by SirYwell**
## Description

`#existing` is e.g. implicitly used by `//replace` with only one argument. It's basically a "not air". It is also used in `#air` but negated, so supporting `InverseMask`s were the original mask can be vectorized is simple and useful.

```[tasklist]
### Submitter Checklist
- [x] Make sure you are opening from a topic branch (**/feature/fix/docs/ branch** (right side)) and not your main branch.
- [x] Ensure that the pull request title represents the desired changelog entry.
- [x] New public fields and methods are annotated with `@since TODO`.
- [x] I read and followed the [contribution guidelines](https://github.com/IntellectualSites/.github/blob/main/CONTRIBUTING.md).
```

### CODE REVIEW

File: worldedit-core/src/main/java/com/fastasyncworldedit/core/internal/simd/SimdSupport.java [hunk 1]
```diff
@@ -44,10 +47,23 @@ public static boolean useVectorApi() {
...
+                }
+                yield (set, get) -> base.compareVector(set, get).not();
+            }
             default -> null;
         };
     }
 
+    private static VectorizedMask vectorizedTargetMaskNonAir() {
+        // everything > VOID_AIR is not air
+        return (set, get) -> get.compare(VectorOperators.UNSIGNED_GT, BlockTypesCache.ReservedIDs.VOID_AIR);
```
Comment (dordsor21) [inline worldedit-core/src/main/java/com/fastasyncworldedit/core/internal/simd/SimdSupport.java:64]: This should just be AIR

File: worldedit-core/src/main/java/com/fastasyncworldedit/core/internal/simd/SimdSupport.java [hunk 2]
```diff
@@ -44,10 +47,23 @@ public static boolean useVectorApi() {
         return switch (mask) {
             case SingleBlockStateMask single -> vectorizedTargetMask(single.getBlockState().getOrdinalChar());
             case InverseSingleBlockStateMask inverse -> vectorizedTargetMaskInverse(inverse.getBlockState().getOrdinalChar());
+            case ExistingBlockMask ignored -> vectorizedTargetMaskNonAir();
+            case InverseMask inverse -> {
+                final VectorizedMask base = vectorizedTargetMask(inverse.inverse());
```
Comment (dordsor21) [inline worldedit-core/src/main/java/com/fastasyncworldedit/core/internal/simd/SimdSupport.java:52]: Any there any/many inverse operations that would be too intensive for this to be efficient?

File: worldedit-core/src/main/java/com/fastasyncworldedit/core/internal/simd/SimdSupport.java (hunk 1 above)
Comment (SirYwell) [reply worldedit-core/src/main/java/com/fastasyncworldedit/core/internal/simd/SimdSupport.java:64]: `ExistingBlockMask` covers all blocks that are considered air, including cave_air and void_air. I'm basically copying that behavior here

Comment (dordsor21) [reply worldedit-core/src/main/java/com/fastasyncworldedit/core/internal/simd/SimdSupport.java:64]: AIr _can_ be void or cave sometimes, but afaik it never is and is always just AIR

File: worldedit-core/src/main/java/com/fastasyncworldedit/core/internal/simd/SimdSupport.java (hunk 2 above)
Comment (dordsor21) [reply worldedit-core/src/main/java/com/fastasyncworldedit/core/internal/simd/SimdSupport.java:52]: Ah yeah I see

File: worldedit-core/src/main/java/com/fastasyncworldedit/core/internal/simd/SimdSupport.java (hunk 1 above)
Comment (SirYwell) [reply worldedit-core/src/main/java/com/fastasyncworldedit/core/internal/simd/SimdSupport.java:64]: cave air is used in structures as it seems. void could probably be ignored, but I don't really want to touch that in any way.

Comment (dordsor21) [reply worldedit-core/src/main/java/com/fastasyncworldedit/core/internal/simd/SimdSupport.java:64]: Mm yeah fair

**Review by dordsor21 — APPROVED** [review]

**Review by NotMyFault — APPROVED** [review]

**Pull request merged**